### PR TITLE
Fix React/Next warnings in console

### PR DESCRIPTION
**Pull request by Tom-Bonnike**

### CODE REVIEW

File: src/components/LoginForm/index.js [hunk 1]
```diff
@@ -37,7 +37,9 @@ const RegistrationForm = () => {
         <Box center>
           <InlineWrapper>
             <Text>First time here ?</Text>
-            <Link href="/register">Create an account</Link>
+            <Link href="/register">
+              <a>Create an account</a>
```
Comment (Tom-Bonnike) [inline src/components/LoginForm/index.js]: I know that’s weird but that’s what Next expects 😄

File: src/components/Loader/index.js [hunk 2]
```diff
@@ -24,3 +24,5 @@ export default styled.div`
     height: 1em;
   }
 `
+
+export default Loader
```
Comment (Tom-Bonnike) [inline src/components/Loader/index.js]: Apparently direct exports like this cause mismatch between server/client.

File: src/pages/_document.js [hunk 3]
```diff
@@ -20,10 +20,11 @@ export default class MyDocument extends Document {
     return (
       <html lang="en">
         <Head>
-          <title>My page</title>
+          <title>Zola App</title>
```
Comment (Tom-Bonnike) [inline src/pages/_document.js]: 🤷‍♂️

Comment (rmnbrd) [inline src/pages/_document.js]: 😄

Comment (Tom-Bonnike) [reply src/pages/_document.js]: :D that was stressing me

**Pull request merged**